File: module-android-api/src/main/java/com/tainzhi/sample/api/adapter/BasicHorizontalAdapter.java
package com.tainzhi.sample.api.adapter;

import androidx.annotation.NonNull;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;
import com.tainzhi.sample.api.R;

import org.jetbrains.annotations.NotNull;

/**
 * Created by muqing on 2019-10-08.
 * Email: qfq61@qq.com
 */
public class BasicHorizontalAdapter extends BaseQuickAdapter<String, BaseViewHolder> {
	public BasicHorizontalAdapter() {
		super(R.layout.item_basic_list);
	}

	@Override
	protected void convert(@NotNull BaseViewHolder baseViewHolder, String s) {
		baseViewHolder.setText(R.id.activity_name, s);
	}
}

File: module-android-api/src/main/java/com/tainzhi/sample/api/adapter/BasicHorizontalAdapter.kt
package com.tainzhi.sample.api.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.tainzhi.sample.api.R

/**
 * Created by muqing on 2019-10-08.
 * Email: qfq61@qq.com
 */
class BasicHorizontalAdapter :
    BaseQuickAdapter<String?, BaseViewHolder>(R.layout.item_basic_list) {
    override fun convert(baseViewHolder: BaseViewHolder, s: String?) {
        baseViewHolder.setText(R.id.activity_name, s)
    }
}